faust/auth.py

```diff
@@ -49,6 +52,33 @@ def __repr__(self) -> str:
         return f"<{type(self).__name__}: username={self.username}>"
 
 
+class OAuthCredentials(Credentials):
+    """Describe OAuth Bearer credentials over SASL"""
+
+    protocol = AuthProtocol.SASL_PLAINTEXT
+    mechanism: SASLMechanism = SASLMechanism.OAUTHBEARER
+
+    ssl_context: Optional[ssl.SSLContext]
+
+    def __init__(
+        self,
+        *,
+        oauth_cb: AbstractTokenProvider,
+        ssl_context: Optional[ssl.SSLContext] = None,
+    ):
```

Comment:
I think `client_id` was supposed to be included in here, based on your test.

Reply:
Thanks for catching this